packages/edit-post/src/style.scss

```diff
@@ -228,10 +229,36 @@ body.block-editor-page {
 	input[type="checkbox"] {
 		border-radius: $radius-round-rectangle / 2;
 
-		&:checked::before {
+		&:checked::before,
+		&[aria-checked="mixed"]::before {
 			margin: -4px 0 0 -5px;
 			color: $white;
 		}
+
+		&[aria-checked="mixed"] {
+			background: theme(toggle);
+			border-color: theme(toggle);
+
+			&::before {
+				// Inherited from `forms.css`.
+				// See: https://github.com/WordPress/wordpress-develop/tree/5.1.1/src/wp-admin/css/forms.css#L122-L132
+				content: "\f460";
+				float: left;
+				display: inline-block;
+				vertical-align: middle;
+				width: 16px;
+				/* stylelint-disable */
+				font: normal 21px/1 dashicons;
```

Comment:
_Really_ minor bug, but it looks like on screens smaller than `782px` (`$break-medium`) wide, checkboxes are scaled up to 25px wide, and this ends up looking tiny: 

![Screen Shot 2019-03-14 at 3 56 46 PM](https://user-images.githubusercontent.com/1202812/54387667-5d79f300-4672-11e9-94a8-9041e1a67776.png)

![Screen Shot 2019-03-14 at 4 01 15 PM](https://user-images.githubusercontent.com/1202812/54387694-6c60a580-4672-11e9-9895-e6771cdab48f.png)
_(Gutenberg doesn't really use max-width breakpoints, so I think that might come from WP-Admin)_

We should add a breakpoint rule to scale that up on smaller screens. So generally, something like: 

```
font: normal 31px/1 dashicons;

@include break-medium() {
    font-size: 21px;
}
```

This'll result in this on small screens: 

![Screen Shot 2019-03-14 at 4 04 29 PM](https://user-images.githubusercontent.com/1202812/54387888-e09b4900-4672-11e9-8f2c-a1e81b393781.png)

... and the appropriate width on larger ones too: 

![Screen Shot 2019-03-14 at 4 05 42 PM](https://user-images.githubusercontent.com/1202812/54387971-0f192400-4673-11e9-839e-caf86f6718e1.png)

---

I'm also seeing slightly better positioning of this pseudo element if we remove the `float: left;` rule on larger screens, so we may want to consider that too. 

![Artboard](https://user-images.githubusercontent.com/1202812/54388305-d7f74280-4673-11e9-82e1-29742c8786a2.png)

Reply:
>Really minor bug, but it looks like on screens smaller than 782px ($break-medium) wide, checkboxes are scaled up to 25px wide, and this ends up looking tiny:

I'll take a look, but noting the prior inline code comment, this is all inherited from existing core stylesheet styles, so if improvements can be made, they should probably be mirrored upstream.

https://github.com/WordPress/wordpress-develop/blob/5.1.1/src/wp-admin/css/forms.css#L122-L132

That said, the mobile styling I believe may be a simple matter that I forgot to transfer over those equivalent styles:

https://github.com/WordPress/wordpress-develop/blob/5.1.1/src/wp-admin/css/forms.css#L1259

Reply:
>That said, the mobile styling I believe may be a simple matter that I forgot to transfer over those equivalent styles:

This should be improved some in 0b45148 . I tried to stay conservative with the adjustments relative to what's inherited from the core forms styling. A tricky problem is that it seems some of the original margins were applied in consideration of only the checkmark font glyph, not the indeterminate stroke.

Reply:
👍 Looks great, thanks @aduth!